src/errorpage.h

```diff
@@ -207,7 +207,7 @@ class ErrorState
     HttpReplyPointer response_;
 
 private:
-    void noteBuildError_(const char *msg, const char *near, const bool forceBypass);
+    void noteBuildError_(const char *, const char *, const bool forceBypass);
```

Comment:
We tend to name parameters in declarations when their meaning cannot be easily guessed (e.g., from the parameter type). This is one of such cases:

```suggestion
    void noteBuildError_(const char *msg, const char *errorLocation, bool forceBypass);
```